project/project/plugins.sbt

```diff
@@ -0,0 +1,2 @@
+
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.5.0")
```

Comment:
Why there?

Reply:
To establish the most recent versions of dependencies (similar functionality to Scala Steward, just from inside sbt shell)

Reply:
There is no need to have 2 `plugins.sbt`

Reply:
Oh, I see what you mean, you ask why it's also in project/project/plugins.sbt and not only in project/plugins.sbt

Reply:
project/project/ folder contains config&plugins specific for the project/ folder itself; so this plugin here can be used to check dependency updates for project/ (i.e. SBT plugin updates), while the same plugin defined in project/plugins.sbt can be used to check "regular" project dependency updates (like scalatest/logback etc.).

Reply:
that's overkill

Reply:
It doesn't harm either... but fair enough, removed (on 3rd round of fixes)